autoinit/estimators/dense.py

```diff
@@ -39,13 +40,45 @@
     'tanh': tfkeras.activations.tanh,
 }
 
+DENSE = tfkeras.layers.Dense
+CONV_1D = (tfkeras.layers.Conv1D, tfkeras.layers.DepthwiseConv1D)
+CONV_2D = (tfkeras.layers.Conv2D, tfkeras.layers.DepthwiseConv2D)
+CONV_3D = tfkeras.layers.Conv3D
+
+DENSE_OR_CONV = (tfkeras.layers.Dense,
+                 tfkeras.layers.Conv1D,
+                 tfkeras.layers.Conv2D,
+                 tfkeras.layers.Conv3D)
```

Comment:
No, I'm just checking whether or not it is a depthwise conv because in that case we need to set `depthwise_initializer` instead of `kernel_initializer`.  Good point though; the code is ambiguous.  I'll make it more clear.